### DISCUSSION TITLE
Understanding Blog Posts

### DISCUSSION THREAD

**Discussion by pclucas14** [alshedivat/al-folio#2030] [Q&A] [answered]
Hi, 

I am trying to understand how to create custom blog posts, and remove the template ones. 

1. Specifically, where is the data for all the sample blog post being stored ?
2. What is the recommended approach to creating a new entry ? I would assume that one can create a markdown file (rather than an html one) ?

Thanks for the great theme!

**Comment by george-gca** [comment, marked as answer]
All blog posts are in [_posts/](https://github.com/alshedivat/al-folio/tree/master/_posts). To remove the posts simply delete these files, and to create new posts create one following the examples there.

**Reply by pclucas14** [reply]
Thank you!
